src/test/java/com/mapzen/pelias/PeliasTest.java

```diff
@@ -41,40 +42,40 @@ public void setup() throws Exception {
     @Test
     public void doc_getDocument() throws Exception {
         peliasWithMock.doc("osmnode", "15", callback);
-        verify(mock).getDoc(eq("osmnode:15"), cb.capture());
+        verify(mock).getDoc(eq("osmnode:15"), eq(apiKey), cb.capture());
     }
 
     @Test
     public void search_getSearch() throws Exception {
-        peliasWithMock.search("test", "1", "2", callback);
-        verify(mock).getSearch(eq("test"), eq("1"), eq("2"), cb.capture());
+        peliasWithMock.search("test", "1", "2", "1", "1", "1", "1", callback);
```

Comment:
It might be helpful to use different min/max values for bounding box so we can test the order is correct.

    peliasWithMock.search("test", "1", "2", "3", "4", "5", "6", callback);